docs/adding_a_feature.md

```diff
@@ -1,39 +1,39 @@
-This is an overview of how we added a feature:
+## Adding a Feature
 
-To make auto-upgrading on the nodes an option.
+###Making auto-upgrading on the nodes an option
```

Comment:
So this documentation is actually an example of adding a feature that exists.  We should probably mention that this is an example.